service-worker.js

```diff
@@ -53,28 +51,21 @@ var regexpRemoveUrlParameters = new RegExp(/([^?#]+)[?#].*$/);
 function removeUrlParameters(url) {
     return url.replace(regexpRemoveUrlParameters, "$1");
 }
-    
-console.log("ServiceWorker startup");
 
 var outgoingMessagePort = null;
 var fetchCaptureEnabled = false;
 self.addEventListener('fetch', fetchEventListener);
-console.log('fetchEventListener set');
 
 self.addEventListener('message', function (event) {
     if (event.data.action === 'init') {
-        console.log('Init message received', event.data);
+        // On 'init' message, we initialize the outgoingMessagePort and enable the fetchEventListener
         outgoingMessagePort = event.ports[0];
-        console.log('outgoingMessagePort initialized', outgoingMessagePort);
         fetchCaptureEnabled = true;
-        console.log('fetchEventListener enabled');
     }
     if (event.data.action === 'disable') {
-        console.log('Disable message received');
+        // On 'idsable' message, we delete the outgoingMessagePort and disable the fetchEventListener
```

Comment:
Presume this is a typo -- should be 'disable' (not 'idsable')

Reply:
That's right, thanks! I'll fix that and merge